src/state-summary/state-card-input_number.html

```diff
@@ -86,14 +113,26 @@
       value: {
         type: Number,
       },
+      mode: {
+        type: String,
+      }
     };
   }
 
+  hiddenState() {
+    if (this.$.slider.offsetWidth < 100) {
+      this.$.sliderstate.hidden = true;
```

Comment:
How does this work? `this.$.sliderstate` is a regular div. What causes `hidden` property to propagate to an attribute?

Reply:
This is my first time with polymer but if `this.$.sliderstate` is a div, isn't `this.$.sliderstate.hidden` the html attribute of the div (and not a property) ?

Reply:
`hidden` is a global attribute: [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/hidden)